### ISSUE TITLE
Comments in verbatim code blocks repeated for every word

### ISSUE THREAD

**Issue by TransientResponse** [copperspice/doxypress#17]
A code line such as this (in Python):
```python
import sys #Make it do work
```

Will become a Latex line like this:
```latex
\textcolor{keyword}{import} os \textcolor{comment}{#Make#Make it#Make it do#Make it do work}
```

**Comment by agserm** [comment]
We have found the root cause for this issue and it will be resolved in the next release of DoxyPress. Thanks for your report!

**Comment by TransientResponse** [comment]
Excellent! I look forward to the next release.

**Comment by agserm** [comment]
The fix for this issue as well as #8 and #16 is in the DoxyPress 1.2.10 release, just posted on our download page. If you could take a look and verify these have been resolved in your test case we would appreciate it.

**Comment by TransientResponse** [comment]
Sorry for getting to this so late; I've been busy at work. Confirmed that 1.2.10 fixes this issue.
